### DISCUSSION TITLE
v3.3.1

### DISCUSSION THREAD

**Discussion by osy** [osy86/HaC-Mini#1083] [Releases]
## Changes (v3.3.1)
* Reverted OpenCore to 0.9.1 release due to `DisableIoMapperMapping` not being needed
* Fixed (really this time) AppleVTD not showing up (#1068)

## Changes (v3.3.0)
* Added new option to disable TRIM in order to improve boot times on certain NVMe devices (#998)
* Added French translation (#1076) (thanks @topiga)
* Added OpenIntelWireless for Ventura (#1077) (thanks @topiga)
* Fixed graphics issues on macOS 13.3 (#1075) (thanks @goodbest)
* Fixed SDXC not detected on macOS 13.2 (#1079) (thanks @topiga)
* Fixed installer failing when unmount fails at the end (#1054)
* Fixed some Thunderbolt devices not being recognized on macOS 13.3 (#1068)

## Driver Update (v3.3.1)
* OpenCore 0.9.2 (OOB) -> 0.9.1

## Driver Updates (v3.3.0)
* AirportBrcmFixup 2.1.6 -> 2.1.7
* AppleALC 1.7.5 -> 1.8.1
* BrcmPatchRAM 2.6.4 -> 2.6.5
* FeatureUnlock 1.0.9 -> 1.1.4
* Lilu 1.6.2 -> 1.6.4
* OpenCore 0.8.5 -> 0.9.2 (OOB)
* Polaris22Fixup 1.3.6 -> 1.3.7
* VirtualSMC 1.3.0 -> 1.3.1
* WhateverGreen 1.6.1 -> 1.6.4

## Messages
* **Important**: Apple has started requiring Secure Boot for updating to macOS Monterey 12.2 and above (including to macOS Ventura). See the [previous release](https://github.com/osy/HaC-Mini/releases/tag/v3.1.5) for details.
* [As announced a few months ago](https://github.com/osy/HaC-Mini/discussions/578), this project will be transitioning to a long-term support model. macOS Ventura will likely be the last supported version.

<hr /><em>This discussion was created from the release <a href='https://github.com/osy/HaC-Mini/releases/tag/v3.3.1'>v3.3.1</a>.</em>

**Comment by topiga** [comment]
Apparently, on macOS 13.3.X, VT-D is a problem. there's two way to fix this issue : the first is with OpenCore 0.9.2, which added a quirk called "DisableIoMapperMapping", to delete Reserved Memory regions. The other way is to have a patched DMAR table. It hasn't been an issue yet, but we should keep an eye on it.
Source : [OpenCorePkg#440](https://github.com/acidanthera/OpenCorePkg/pull/440) and [OpenCorePkg#451](https://github.com/acidanthera/OpenCorePkg/pull/451)

**Reply by osy** [reply]
Did you experience this problem?

**Reply by topiga** [reply]
A long time ago, that's why I already had the patched DMAR and DMAC. My issue was with bluetooth I think. I don't know if this is what solved the issue, I did multiple things.

**Reply by osy** [reply]
Okay, please test the release and see if the issue arises instead of just assuming that something is an issue, thanks.

**Reply by osy** [reply]
(If you look at the change-log v3.3.0 had 0.9.2 + DisableIoMapperMapping and it was useless. Plus @SashonIT confirmed that it worked on 13.3.)

**Reply by topiga** [reply]
> It hasn't been an issue yet, but we should keep an eye on it.

I know ^^
I'm not saying it is an issue, but I said it might be. The description of the problem is that all Intel machines with an iGPU, 16GB of RAM or more are affected. I'm not assuming that it is an issue, it just picked my interest, that's all

**Reply by topiga** [reply]
Btw I don't have the Intel card anymore, and I'm not on macOS 13.3 for various reason, so even if it's still an issue, I can't test it.

**Reply by vtungn** [reply]
I have both wifi and bluetooth not working. I use Apple Broadcom card

**Reply by topiga** [reply]
What kexts are enabled ?
